### PR TITLE
i18n(ko-KR): update `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `configuration-reference.mdx`
- #6589

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -278,50 +338,33 @@ Astro 컴포넌트의 요소 선택기가 전역 스타일 기본값 (예: 전
 ``where'`를 사용하면 구체성을 더 잘 제어할 수 있지만, 적용되는 선택기를 제어하려면 더 높은 구체성 선택기, 레이어 및 기타 도구를 사용해야 합니다.
 `'attribute'`를 사용하는 것은 요소의 `class` 속성을 조작하고 자신의 스타일 지정 로직과 Astro의 스타일 적용 사이의 충돌을 피해야 할 때 유용합니다.
 
-### 어댑터
-
-<p>
+### vite
 
-**타입:** `AstroIntegration`
-</p>
```
Comment (dreyfus92) [inline src/content/docs/ko/reference/configuration-reference.mdx:286]: This needs to be added, according to this commit 68457524dcea899c61b8b20ae865db7e35dfe74a I would've linked the exact part but I was unable to link it for some reason.
```suggestion
**타입:** `ViteUserConfig`
</p>
```

**Review by dreyfus92 — COMMENTED** [review]
Amazing job @jsparkdev, I left you a small suggestion 😁

**Comment by jsparkdev** [comment]
thank you Added missing type!

**Review by dreyfus92 — APPROVED** [review]
Thank you @jsparkdev, LGTM 😁

**Pull request merged**